### ISSUE TITLE
SSI application on paid/volunteer contributions

### ISSUE THREAD

**Issue by EstherPlomp** [the-turing-way/the-turing-way#3833]
### Summary

Esther is sending in an application for the SSI fellowship programme 2025 to look at differences between paid/volunteer members, their contributions and their dynamics within The Turing Way Community. 
- [Slides](https://docs.google.com/presentation/d/1HukcKxB9XtFLy70sVvjHyjo968ja7qjy/)
- [Recording](https://tudelft.zoom.us/rec/share/-h-ApUjSSGou8_1Gha9QL1fH3b1gAyL7gQ-FzqUq-uO-2y97ZXkb3zfJ7wDuwHSc.t9U-z15a3hTc9bfz)

### What needs to be done?

- [ ] Submit application
- [ ] Whether submitted or not some of the information below can be relevant to continue the project

### Who can help?

* If selected: hopefully anyone in the community that wants to fill out a survey or do a short in depth interview!

---

### Resources & Info
- **Sun et al. 2024**
- If Delphi: The first two rounds of the Delphi were online surveys administered using Surveylet. Surveylet
is a purpose-built platform for developing and administering Delphi surveys [19]. To
start with, the Delphi participants were presented with an initial set of 17 potential open science
practices to consider that were generated by the project team based on a discussion.
Round 3 took the form of two half-day meetings hosted on Zoom [20]. **Cobey et al. 2023**
- For Delphi see also: https://www.youtube.com/watch?v=ViQ0fylL0Ig
- **Jensen and Scacchi 2005** -> research is focusing more on corporate/private and volunteer collaborations instead of a general distinction between paid and non-paid contributors
- **Sharma et al. 202**2 contributions: volunteer management remains an overlooked aspect not only in the management research in general (Ertas, 2019).
Finally, research shows that when volunteers and paid staff work together in a volunteer-involving organization (e.g., nonprofits), they often perceive the organization’s values through distinct personal lenses, leading to disagreements (Kreutzer & Jäger, 2011). When unpaid developers perform the same coding tasks as paid developers equally well, it is natural for paid developers to feel threatened and for unpaid developers to feel underappreciated. The Debian project highlights that similar issues arise in the OSS context as well (Gerlach et al., 2016) and that management processes need to be developed to mitigate such issues. NGOs often employ volunteer resource managers who help oversee issues in maintaining relationships between paid staff and volunteers (Ertas, 2019). Can certain individuals act as liaisons between volunteer and paid OSS developers to manage expectations and interactions better?
- James Howison - PhD candidate researching paid roles in Oss vs unpaid

**Comment by Arielle-Bennett** [comment]
Good luck for the application Esther!

**Comment by EstherPlomp** [comment]
Thank you @Arielle-Bennett! You as well!

**Comment by BatoolMM** [comment]
I would be interested in learning more about your work, @EstherPlomp. I will be joining Collaboration Cafe on 19 March

**Comment by penyuan** [comment]
Good luck @EstherPlomp!!

Probably not mission critical, but FYI I've been looking at QualCoder as a possible open source replacement for NVivo for qualitative coding: 

https://github.com/ccbogel/QualCoder

It's been in active development for years and comes with comprehensive user documentation.

**Comment by EstherPlomp** [comment]
thanks @penyuan ! I actually followed a QualCoder workshop last week and it looks like it has everything that I need and indeed no proprietary drama - so I will most likely use this solution indeed!

**Comment by EstherPlomp** [comment]
Survey Questions: 

## Survey information
You are being asked to participate in a research study conducted by Esther Plomp on behalf of The Turing Way for Esther’s SSI Fellowship. This survey is intended to identify what The Turing Way community members consider as ‘being a paid contributor’, so that we can better understand how contributions to the book and community are facilitated, as well as any resulting dynamics within the community. 

Participation in the research is voluntary, and the decision to participate or not will have no impact on your relationship with The Turing Way community. For the full ethical procedures of this study, please see: https://hackmd.io/BLUnM20cS1iv5-JiL7-k9g. For any questions, please contact Esther (esther.plomp@ua.aw).

## Section 1: Contributing paid or voluntarily 
**In what way do you consider yourself to be a Turing Way community member? (Tick all that apply)**
•	Yes, I engage with discussions on Slack
•	Yes, I have contributed to the book via GitHub
•	Yes, I have participated in events, such as Community Calls, Collaboration Cafés or a Book Dash
•	Yes, I took leadership roles in a working group or in the organisation of an event
•	No, I haven't engaged with The Turing Way at all before opening up this survey

**Do you think that your contributions to The Turing Way are adequately recognised and rewarded? Please elaborate on how your contributions are recognised (or not) by both The Turing Way Community and your employer (if this is relevant).** 

**In what capacity are you normally contributing to the Turing Way?** _Are you in any way compensated for your time (such as receiving feedback on your work, you are allowed to work on contributions during working hours, contributing to the Turing Way allows you to develop skills or build up CV activities that other work activities do not)._ 
-	On a volunteer basis: my contributions have nothing to do with my current work position and I contribute in my free time
-	On a volunteer basis: my contributions are related to my work position but I do not get time allotted to contribute to The Turing Way and I therefore contribute in my free time
-	On a paid basis: my contributions are related to my work and there is (formally) dedicated work time for this
-	On a paid basis: I receive funding or financial support, such as a research grant, that facilitates my contributions to The Turing Way
-	I’m not sure

**Can you elaborate further on your previous answer?** 

**Approximately how often do you normally contribute to The Turing Way?**
_A contribution may include participation in a discussion on Slack, being active on the GitHub repository (by opening of an Issue/PR or commenting on an existing one), or attending an event (such as a Collaboration Café or Fireside chat)._ 
•	Daily
•	Several times a week
•	Once a week
•	Several times a month
•	Once a month
•	Several times a year
•	Annually
•	Never
•	Other (please describe)

## Section 2/ Wrap up and possible follow up for semi-structured interview.

**Please indicate here if you consent to being contacted in the future to discuss your answers to the survey in more detail**
Yes - I have provided my email above
No

**Who can you remember that isn't around anymore that would want to provide their input?**

**Comment by EstherPlomp** [comment]
Interview questions: 

**Optional demographic data**
•	Country currently based in
•	Gender

**Non optional data:**
•	**Volunteer/paid contributor** _What do you see as ‘being paid’?_
•	**Tell me about your involvement with the Turing Way community?** _Why do you contribute? What got you interested in the community? What has kept you engaged? What has been difficult when trying to contribute to The Turing Way?_

**•	Do you feel adequately compensated/positively impacted for your efforts in The Turing Way?** _Could you tell me more about that (when, how it happened, what it entailed - such as positive impact career/development and so forth). W hy do you see that as a success? Why do you see it as a failure? What do you see as adequate compensation?_ 

**•	Do you feel like it makes a difference to be a volunteer or paid member to contribute to the Community?** _Does it make a difference in leadership roles?_

**•	How does paid contribution relate to the sustainability of the community?**

**•	What could the Turing Way do to facilitate contributions to the community?** _How can community development be improved?What does the community need more of?_

**Comment by penyuan** [comment]
Hi @EstherPlomp! I'm following up here from your Slack post. 

Some minor comments: 

1. I feel the "yes"s and "no"s could be removed from the first survey question on "In what way do you consider yourself..." because it's not worded as a yes/no question. 
2. It may complicate analysis, but I feel the way all survey questions are currently framed means they could all have "other, please describe" options (instead of only the last question about how often you contribute). 
3. For the "In what capacity are you normally contributing..." survey question, it comes across as two questions. One about whether you're being paid for your contributions, the other about to what degree your contributions are part of your official work. Depending on how you want to analyse the responses, it may (of may not??) be helpful to split it into two questions. 
4. For the same question, consider adding "please explain" to the "I’m not sure" option. 
5. Pedantic thought: I can imagine someone struggling with this question ("In what capacity are you normally contributing...") because some people consider an activity to be volunteer work if they do it in their "free time" *even if* they are being paid for the volunteer work, while other people might consider "paid volunteering" to be an oxymoron because if you're being paid then you're not volunteering. The point being people may have different imaginations of how "volunteering" is delineated/defined (e.g., paid vs unpaid/"free time" vs "work time"/etc.). I don't know how this should influence your survey and interview design, but just a thought..... I suspect you've probably considered this already! I also know you get into this in the interview, but there might be people who only complete the survey, so the question is do you want to get into this nuance there (or not)...
6. For "Please indicate here if you consent to being contacted in the future...", are you intentionally leaving it open or should you specify that it is to be possibly followed up with a semi-structured interview? If it's the latter, then I suggest givng a sense of the time commitment. But again, maybe better to leave this more open-ended...
7. For the interview demographic data, I suggest asking "Where in the world are you currently based in" unless you specifically want to ask about "country" as a political entity. 
8. Asking more out of curiosity: How much time do you plan to spend on each interview? And how many people do you hope to interview?

Sorry I can't make the Collaboration Cafe on 21 January 2026, but hope some of this is helpful. Thank you for doing this work! 🙏

**Comment by EstherPlomp** [comment]
Thanks @penyuan! I'm adjusting as you suggested. The idea is to interview 10-20 community members with a spread in whether they consider themselves to be volunteer or paid community members / different distribution in activity, with interviews taking around 30-90 minutes (pending on how opinionated people are :))
